baserun/helpers.py

```diff
@@ -14,4 +19,28 @@ class BaserunType(Enum):
 class BaserunStepType(Enum):
     LOG = auto()
     AUTO_LLM = auto()
-    CUSTOM_LLM = auto()
\ No newline at end of file
+    CUSTOM_LLM = auto()
+
+
+def baserun_threaded_wrapper(
```

Comment:
Consider moving this to baserun.py as a baserun.thread() or baserun.thread_wrapper() function  and expose via the __init__.py